feature-settings-impl/src/main/java/io/novafoundation/nova/feature_settings_impl/presentation/cloudBackup/BackupSettingsViewModel.kt

```diff
@@ -147,6 +187,52 @@ class CloudBackupSettingsViewModel(
 
         isSyncing.value = false
     }
+
+    private fun manageBackupChangePasswordItem(): ListSelectorMixin.Item {
+        return ListSelectorMixin.Item(
+            R.drawable.ic_pin,
+            R.color.icon_primary,
+            R.string.common_change_password,
+            R.color.text_primary,
+            ::onChangePasswordClicked
+        )
+    }
+
+    private fun manageBackupDeleteBackupItem(): ListSelectorMixin.Item {
+        return ListSelectorMixin.Item(
+            R.drawable.ic_delete,
+            R.color.icon_negative,
+            R.string.backup_settings_delete_backup,
+            R.color.text_negative,
+            ::onDeleteBackupClicked
+        )
+    }
+
+    private fun onChangePasswordClicked() {
+        TODO()
```

Comment:
I suggest to mark todo functionality with showMessage("TODO") in future - otherwise while testing and clicking on yet unsuppored button the app crashes, which is a bit unconcvnient